src/components/OverflowMenuItem/OverflowMenuItem.js

```diff
@@ -115,6 +117,11 @@ OverflowMenuItem.propTypes = {
    * `true` if this menu item belongs to a floating OverflowMenu
    */
   floatingMenu: PropTypes.bool,
+
+  /**
+   * `true` if this menu item has long text and requires a browser tooltip
+   */
+  title: PropTypes.bool,
```

Comment:
Just a nitpick - Wondered if we can change the prop name to `requireTitle` or something... I naïvely thought `title` goes directly to the tooltip when I saw the new prop name.

Reply:
Yeah after thinking about it this weekend, I agree that it should be a bit more semantic. Will update